src/components/whats-new.vue

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <modal :state="isVisible" backdrop :hideable="true" @hide="hideModal">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <div class="modal-introduction">
```

Comment:
```suggestion
      <p class="modal-introduction">
```

We make the `.modal-introduction` element a `<div>` when it's wrapping multiple `<p>` elements, but there's always some `<p>`.